packages/ci-v5/CHANGELOG.md

```diff
@@ -816,7 +816,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [1.5.2] - 2017-04-27
 ### Changed
-- Releax engines requirements in package.json.
+- Relax engines requirements in package.json.
```

Comment:
Depending on things, it's possible that `engine` is correct (but my guess is that the current word is the right one)